src/components/consul/selectors/DocumentSelector.js

```diff
@@ -1,28 +1,66 @@
 import PropTypes from 'prop-types';
 import React, { useEffect, useState } from 'react';
-import { TouchableOpacity, View } from 'react-native';
+import { useMutation } from 'react-apollo';
+import { Alert, TouchableOpacity, View } from 'react-native';
 
 import { colors, Icon, normalize, texts } from '../../../config';
+import { ConsulClient } from '../../../ConsulClient';
 import { deleteArrayItem, documentErrorMessageGenerator, formatSize } from '../../../helpers';
 import { useSelectDocument } from '../../../hooks';
+import { DELETE_DOCUMENT } from '../../../queries/consul';
 import { Button } from '../../Button';
 import { Input } from '../../form';
 import { RegularText } from '../../Text';
 import { WrapperRow } from '../../Wrapper';
 
+const deleteDocumentAlert = ({ onDeleteDocument, documentId, index }) =>
+  Alert.alert(
+    texts.consul.startNew.deleteAttributesAlertTitle,
+    texts.consul.startNew.documentDeleteAlertBody,
+    [
+      {
+        text: texts.consul.abort,
+        onPress: () => null,
```

Comment:
is passing a method returning `null` necessary here for doing "nothing" on press? what if `onPress` would not be given?